docs/languages/csharp-properties.md

```diff
@@ -0,0 +1,253 @@
+# Introduction
+Properties are first class citizens in C#. The language
+defines syntax that enables developers to write code
+that accurately expresses their design intent.
+
+Properties behave like fields when they are accessed.
+However, unlinke fields, properties are implemented
+with accessors that define the statements executed
+when a property is accessed or assigned.
+
+# Property Syntax
+The syntax for properties is a natural extension to
+fields. A field defines a storage location:
+
+```
+public class Person
+{
+	public string FirstName;
+	// elided
```

Comment:
I had to look up "elided" - not a common term. Repeated in numerous code examples. Do we even need a comment here, and has the reader already been introduced to the "//" comment syntax?